rest/model/subscriber.go

```diff
@@ -163,6 +192,40 @@ func (s *APIGithubPRSubscriber) ToService() (interface{}, error) {
 	}, nil
 }
 
+func (s *APIGithubMergeSubscriber) BuildFromService(h interface{}) error {
+	if v, ok := h.(event.GithubMergeSubscriber); ok {
+		h = &v
+	}
+
+	switch v := h.(type) {
+	case *event.GithubMergeSubscriber:
+		s.Owner = ToAPIString(v.Owner)
+		s.Repo = ToAPIString(v.Repo)
+		s.PRNumber = v.PRNumber
+		s.Ref = ToAPIString(v.Ref)
+		s.CommitMessage = ToAPIString(v.CommitMessage)
+		s.MergeMethod = ToAPIString(v.MergeMethod)
+		s.CommitTitle = ToAPIString(v.CommitTitle)
+
+	default:
+		return errors.New("unknown type for APIGithubMergeSubscriber")
```

Comment:
`errors.Errorf("unable to handle type '%T'", v)`

Reply:
Done.